torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4245,19 +4285,43 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    ref=np.i0,
                    decorators=(precisionOverride({torch.bfloat16: 3e-1,
                                                   torch.float16: 5e-1}),),
+                   skips=(
+                       SkipInfo('TestOpInfo', 'test_supported_backward',
```

Comment:
We now have new metadata:

https://github.com/pytorch/pytorch/blob/1461859fdebe1346accf50751854b5448b832be8/torch/testing/_internal/common_methods_invocations.py#L161

which avoids these skips

Reply:
Note: 
Would be nice we if could just pass `backward_dtypes` for UnaryUfunc,
However with the way, `None` is handled for `backward_dtypesIf*`, for using it with `UnaryUfunc` we have to specify all of them as UnaryUfunc specifies all `dtypesIf*`.
https://github.com/pytorch/pytorch/blob/1fc89d9ffcea234ccf9a35f60a7afc3cb0d406ee/torch/testing/_internal/common_methods_invocations.py#L210-L216

https://github.com/pytorch/pytorch/blob/1fc89d9ffcea234ccf9a35f60a7afc3cb0d406ee/torch/testing/_internal/common_methods_invocations.py#L385-L387

Reply:
Yes but we should fix that so it has the same logic as a regular OpInfo

Reply:
Noted :)